### DISCUSSION TITLE
m4b files unknown duration + no tracks + no picture

### DISCUSSION THREAD

**Discussion by masterwishx** [kagemomiji/airsonic-advanced#247] [Q&A] [answered]
Im very sorry for posting another problem ,but found errors with m4b files.

it can be played fine from start but no tracks in it and no picture , also cant be forwarded.

![image](https://github.com/kagemomiji/airsonic-advanced/assets/28630321/4d884a6f-791b-4850-9dae-2742a1c38400)

![image](https://github.com/kagemomiji/airsonic-advanced/assets/28630321/3973a667-2466-43d6-990a-32ee55517553)

![image](https://github.com/kagemomiji/airsonic-advanced/assets/28630321/aac95db8-f563-4b24-bc3a-ec3cb30c0056)

![image](https://github.com/kagemomiji/airsonic-advanced/assets/28630321/145f7b1d-3e40-4dec-8310-ba86a5254c50)

**Comment by masterwishx** [comment]
![image](https://github.com/kagemomiji/airsonic-advanced/assets/28630321/9162dfb0-5a25-4278-94b8-2826487f615d)

**Comment by luk6173** [comment]
I also don't see any pictures from my MP3 files, seems to be a bug?

**Reply by masterwishx** [reply, marked as answer]
In my case it was mb4 file problem, new files fixed issue, also mb4 now is working and fixed in latest releases

**Comment by luk6173** [comment]
> new files fixed issue

new files?

**Reply by masterwishx** [reply]
When I uploaded new mb4 file instead

**Reply by luk6173** [reply]
do you see your pictures of your tracks?
